pkg/client/query/accquerier.go

```diff
@@ -51,6 +53,9 @@ func (aq *accQuerier) GetAccount(
 	ctx context.Context,
 	address string,
 ) (types.AccountI, error) {
+	aq.accountCacheMu.Lock()
+	defer aq.accountCacheMu.Unlock()
+
```

Comment:
#PUC: Why do we need to protect against concurrent reads to the account cache?

Reply:
TL;DR; golang complaining about concurrent access during the load test.

Since `GetAccount` is doing both reads and writes to the cache map, and having a rpc query in between cache reads and writes; a second call to the method (with the same address) trying to read from the cache could be concurrent with a write initiated by a first call.